### PR TITLE
Fix Pics Statement Failing Unsupported Properties

### PR DESCRIPTION
**Pull request by jhughesbiot**
Corrects issue: https://github.com/faucetsdn/daq/issues/886

Properties indicated as not supported, will be shown as SKIPPED in the result.  The same will occur if an object is in the pics statement but is not in the device as long as all properties are marked false.

Tested against the most recent device that had this failure present itself to confirm both test builds and real world scenarios.  Image included to show how the report and pics statement handle these scenarios.

image

### CODE REVIEW

**Review by grafnu — APPROVED** [review]

**Comment by jhughesbiot** [comment]
@grafnu 
Is there any way to get a report of the stickler-ci failure for a single file?  I'd like to get that issue resolved before merging just not seeing anyway to run the formatting check against a single file and get a report.

**Comment by grafnu** [comment]
I don't know how to do that specifically (single file), but if you click
through on the detail of the failure it takes you to a summary page:
https://github.com/faucetsdn/daq/pull/948/checks?check_run_id=5192330626

Looks like there's an entry in .checkstyle.xml that's tripping it up. Not
sure what changed (likely a newer version of stickler checks...)

**Comment by jhughesbiot** [comment]
I think I have a fix for the stickler but would want to get this merged in first so I can get a clean PR going to just address that issue.  I can't merge it in since I don't have write access.

**Pull request merged**
